build/dependencies.yaml

```diff
@@ -1,4 +1,48 @@
 dependencies:
+  - name: "cni"
+    version: 0.8.5
+    refPaths:
+    - path: build/workspace.bzl
+      match: CNI_VERSION =
+    - path: cluster/gce/gci/configure.sh
+      match: DEFAULT_CNI_VERSION=
+    - path: cluster/gce/config-common.sh
+      match: WINDOWS_CNI_VERSION=
+    - path: test/e2e_node/remote/utils.go
+      match: cniVersion[\t\n\f\r ]*=
+
+  - name: "coredns-kube-up"
+    version: 1.6.7
+    refPaths:
+    - path: cluster/addons/dns/coredns/coredns.yaml.base
+      match: k8s.gcr.io/coredns
+    - path: cluster/addons/dns/coredns/coredns.yaml.in
+      match: k8s.gcr.io/coredns
+    - path: cluster/addons/dns/coredns/coredns.yaml.sed
+      match: k8s.gcr.io/coredns
+
+  - name: "coredns-kubeadm"
+    version: 1.6.7
+    refPaths:
+    - path: cmd/kubeadm/app/constants/constants.go
+      match: CoreDNSVersion =
+
+  - name: "crictl"
+    version: 1.17.0
```

Comment:
ooh this should get updated

Reply:
Yep, yep. There's an update PR in-flight (which needs approval) here: https://github.com/kubernetes/kubernetes/pull/89720

Reply:
FWIW kind ships 1.18 and we're doing fine so far :-)

Reply:
on it